src/com/stag/horns/fragments/StatusbarSettings.java

```diff
@@ -67,4 +86,55 @@ public void onPause() {
     public boolean onPreferenceChange(Preference preference, Object newValue) {
```

Comment:
Remove this duplicate boolean as you created a new one below

Reply:
Done :D